src/domFacade/ConcreteNode.ts

```diff
@@ -9,7 +9,7 @@ import {
 	Text,
 } from '../types/Types';
 
-export const enum NODE_TYPES {
+export enum NODE_TYPES {
```

Comment:
Why did you make this change? the `export const enum` causes it to be inlined, I think

Reply:
Yes, I did it so generated code could use it from the runtime lib. However, after you mentioned it I went looking for another way to do it and found that inlining the enum value in the generated code also works (which is better). Fixed it.